subsys/bluetooth/mesh/net.c

```diff
@@ -63,7 +63,7 @@
 
 static struct {
 	uint32_t src : 15, /* MSb of source is always 0 */
-	      seq : 17;
+			 seq : 17;
```

Comment:
Please don't mix coding style cleanups into a patch with functional changes (you can still have them in the same PR but in separate commits)